spyder/plugins/variableexplorer/widgets/namespacebrowser.py

```diff
@@ -219,6 +228,7 @@ def setup_options_button(self):
         if not self.options_button:
             self.options_button = create_toolbutton(
                 self, text=_('Options'), icon=ima.icon('tooloptions'))
+            self.options_button.setPopupMode(QToolButton.InstantPopup)

```

Comment:
Why is this necessary?

Reply:
I think is not necessary, this line probably got there when I was checking the `options_button` for the `Object Explorer`. I will remove it :+1: